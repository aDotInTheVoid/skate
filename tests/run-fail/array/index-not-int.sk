fn main() {
    [][2.9];
}
