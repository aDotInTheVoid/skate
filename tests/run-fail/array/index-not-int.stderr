error: Expected `int`, got `float`
  ┌─ $DD/tests/run-fail/array/index-not-int.sk:2:8
  │
2 │     [][2.9];
  │        ^^^ Evaluated to `Float(2.9)`

